src/resources/locales/en.json

```diff
@@ -174,18 +177,18 @@
         },
         "text": "Text",
         "left": "Left",
-        "justify": "Justified",
+        "justify": "Justify",
         "column": {
           "title": "Column",
           "auto": "Automatic",
           "one": "1 column",
-          "two": "2 columns"
+          "two": "2 column"
         },
         "spacing": "Spacing",
         "margin": "Margin",
-        "wordSpacing": "Word spacing",
+        "wordSpacing": "Words spacing",
         "lineSpacing": "Line spacing",
-        "letterSpacing": "Letter spacing",
+        "letterSpacing": "Letters spacing",
```

Comment:
Bad move. I don't think it is plural. @danielweck ?

Reply:
I think singular works better, both as a technical term and as a human-readable label.
https://developer.mozilla.org/en-US/docs/Web/CSS/word-spacing
https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing